css/style.css

```diff
@@ -449,6 +449,95 @@ h3.lang-1 {
   right: 80px;
 }
 
+.contact-form {
+  text-align: center;
+  border-top-left-radius: 80px 80px;
+  padding: 103px 24px 45px;
+  background: #6070ff no-repeat right/60%  url(../img/contact-form-shapes/contact\ form\ shapes-mobile@2x.png);
+  flex-shrink: 0;
+}
+
+.contact-form h2 {
+  font-family: Poppins, sans-serif;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 52px;
+  color: #fff;
+  padding-bottom: 12px;
+}
+
+.contact-form .contact-paragraph {
+  text-align: center;
+  font-family: Poppins-Thin, sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  color: #ebebff;
+  padding-left: 24px;
+  padding-right: 24px;
+  padding-bottom: 52px;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.input {
+  border: 0;
+  padding: 5%;
+  border-radius: 12px;
+  font-family: Poppins, sans-serif;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 20px;
+  color: #172b4d;
+}
+
+.input-text {
+  border: 1px solid #cfd8dc;
+  padding: 5%;
+  height: 170px;
+  flex-shrink: 0;
+  border-radius: 8px;
+  font-family: Poppins, sans-serif;
+  color: #b3bac5;
+  padding-bottom: 22px;
+  width: 100%;
+}
+
+.button-contact-form {
+  display: inline-flex;
+  align-items: center;
+  margin-top: 12px 16px;
+  margin-right: 209px;
+  padding: 12px 16px;
+  width: 142px;
+  height: 48px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  background: #f7f7f9;
+  color: #6070ff;
+  font-size: 17px;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.51px;
+  padding-bottom: 45px;
+  cursor: pointer;
+}
```

Comment:
- [`OPTIONAL`] Kindly consider adjusting the padding of your `button` element to match the design in Figma. Upon careful inspection, I noticed that there is excessive padding at the bottom of the button, causing it to deviate from the Figma design. To resolve this issue, please remove the `padding-bottom: 45px` property, as it appears to be the main cause of the mismatch. By making this adjustment, you should be able to align the button with the desired design. See below for more details:

<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="160" alt="Screenshot 2023-07-12 at 16 30 11" src="https://github.com/IvonneBenitesRodriguez/microverse_mobile_portfolio/assets/107192076/1ecfba9a-78d0-4b95-8a2c-872db47ee5cb">
</td>
<td><img width="160" alt="Screenshot 2023-07-12 at 16 21 37" src="https://github.com/IvonneBenitesRodriguez/microverse_mobile_portfolio/assets/107192076/b06b6486-f888-40ed-9ce0-c363dbb42a1c"></td>
</tr>
</table>